pravega/pravega
# Issue 5258: Spotbugs HTML Reports.

**Change log description**  

* Enable HTML report generation and disable XML report generation.

**Purpose of the change**  
#Fixes #5258 

**What the code does**  

Adds configuration to `spotbugs` tasks.

**How to verify it**  
(Optional: steps to verify that the changes are effective)

Run a `./gradlew build` and see the HTML report.
